moto/ec2/responses/transit_gateway_route_tables.py

```diff
@@ -139,6 +153,13 @@ def search_transit_gateway_routes(self):
         <destinationCidrBlock>{{ transit_gateway_route_table.routes[destination_cidr_block]['destinationCidrBlock'] }}</destinationCidrBlock>
         <state>{{ transit_gateway_route_table.routes[destination_cidr_block]['state'] }}</state>
         <type>{{ transit_gateway_route_table.routes[destination_cidr_block]['type'] }}</type>
+        <transitGatewayAttachments>
+            <item>
+                <resourceId>{{ transit_gateway_route_table.routes[destination_cidr_block]['transitGatewayAttachments']['resourceId'] }}</resourceId>
```

Comment:
Can we safely assume that `...['transitGatewayAttachments']['resourceId']` will always be present, or should we hedge against `KeyError` here?
```
`....get('transitGatewayAttachments', {}).get('resourceId')`
```